server/config/conditions.ts

```diff
@@ -677,6 +678,62 @@ export default {
           label: 'Enter name of approved premises',
           name: 'approvedPremises',
           case: 'capitalised',
+          listType: 'AND',
+          addAnother: {
+            label: 'Add another approved premises',
+          },
+        },
+        {
+          type: InputTypes.TIME,
+          label: 'Enter a reporting time',
+          name: 'reportingTime',
+        },
+        {
+          type: InputTypes.RADIO,
+          label: 'Select a review period',
+          name: 'reviewPeriod',
+          case: 'lower',
+          options: [
+            {
+              value: 'Weekly',
+            },
+            {
+              value: 'Monthly',
+            },
+            {
+              value: 'Other',
+              conditional: {
+                inputs: [
+                  {
+                    type: InputTypes.TEXT,
+                    label: 'Enter a review period',
+                    name: 'alternativeReviewPeriod',
+                  },
+                ],
+              },
+            },
+          ],
+        },
+      ],
+      type: ReportToApprovedPremises,
+    },
+    {
+      code: '2027ae19-04a2-4fa6-8d1b-a62dffba2e62',
+      category:
+        'Supervision in the community by the supervising officer, or other responsible officer, or organisation',
+      text: 'Report to staff at [NAME OF POLICE STATION] at [TIME / DAILY], unless otherwise authorised by your supervising officer. This condition will be reviewed by your supervising officer on a [WEEKLY / MONTHLY / ETC] basis and may be amended or removed if it is felt that the level of risk you present has reduced appropriately.',
+      tpl: 'Report to staff at {policeStation} at {reportingTime}, unless otherwise authorised by your supervising officer. This condition will be reviewed by your supervising officer on a {reviewPeriod} {alternativeReviewPeriod} basis and may be amended or removed if it is felt that the level of risk you present has reduced appropriately.',
+      requiresInput: true,
+      inputs: [
+        {
+          type: InputTypes.TEXT,
+          label: 'Enter name of police station',
+          name: 'policeStation',
+          case: 'capitalised',
+          listType: 'AND',
+          addAnother: {
```

Comment:
Is policeStation a list or single data item? If single, could lose the listType: AND.

Reply:
It does look like a list in config/validation, so ignore this one ^